### PR TITLE
added slideshow content kind and necessary format presets

### PR DESCRIPTION
**Pull request by nucleogenesis**

### CODE REVIEW

File: le_utils/constants/content_kinds.py [hunk 1]
```diff
@@ -32,6 +34,7 @@
     file_formats.EPUB: DOCUMENT,
     file_formats.PERSEUS: EXERCISE,
     file_formats.HTML5: HTML5,
+    file_formats.SLIDESHOW: SLIDESHOW,
```
Comment (ivanistheone) [inline le_utils/constants/content_kinds.py]: TODO: discuss

File: le_utils/constants/file_formats.py [hunk 2]
```diff
@@ -62,6 +62,8 @@
 EPUB = "epub"
 EPUB_MIMETYPE = "application/epub+zip"
 
+# constants for Slideshow format
+SLIDESHOW = "slideshow"
```
Comment (ivanistheone) [inline le_utils/constants/file_formats.py]: DOn't need since alraedy have GIF, PNG, and JPG, JPEG

File: le_utils/constants/content_kinds.py (hunk 1 above)
Comment (rtibbles) [reply le_utils/constants/content_kinds.py]: I feel like `IMAGE` might be better here, and doesn't tie us too tightly to our implementation choices.

File: le_utils/constants/file_types.py [hunk 3]
```diff
@@ -10,6 +10,7 @@
 HTML5 = "html5"
 THUMBNAIL = "thumbnail"
 SUBTITLES = "subtitles"
+SLIDESHOW = "slideshow_image"
```
Comment (jayoshih) [inline le_utils/constants/file_types.py]: Would it make sense to either call the variable `SLIDESHOW_IMAGE` or the value to be `slideshow` to keep the mapping more consistent?

File: le_utils/resources/formatlookup.json [hunk 4]
```diff
@@ -40,5 +40,8 @@
 	},
 	"zip": {
 		"mimetype": ".zip"
+	},
+	"slideshow_manifest": {
```
Comment (jayoshih) [inline le_utils/resources/formatlookup.json]: Is there a reason we're not just using the json file format for this?

File: le_utils/constants/file_types.py (hunk 3 above)
Comment (ivanistheone) [reply le_utils/constants/file_types.py]: +1 for change constant name from `SLIDESHOW` to `SLIDESHOW_IMAGE`

File: le_utils/constants/format_presets.py [hunk 5]
```diff
@@ -54,6 +54,15 @@
 HTML5_DEPENDENCY_ZIP = "html5_dependency"
 HTML5_DEPENDENCY_ZIP_READABLE = "HTML5 Dependency (Zip format)"
 
+SLIDE_IMAGE = "slide_image"
```
Comment (ivanistheone) [inline le_utils/constants/format_presets.py]: @indirectlylit suggested changing the names `SLIDE_IMAGE` and `slide_image` to 
`SLIDESHOW_IMAGE` and `slideshow_image`

@nucleogenesis I let you see what makes more sense

File: le_utils/resources/presetlookup.json [hunk 6]
```diff
@@ -226,5 +226,41 @@
...
+		"convertible_formats": []
+	},
+	"slideshow_thumbnail": {
+		"readable_name": "Slideshow Thumbnail",
+		"multi_language": false,
+		"supplementary": true,
+		"thumbnail": true,
+		"subtitle": false,
+		"display": false,
+		"order": 3,
```
Comment (ivanistheone) [inline le_utils/resources/presetlookup.json]: @nucleogenesis 
Could you please change that to `2` here

File: le_utils/resources/presetlookup.json [hunk 7]
```diff
@@ -226,5 +226,41 @@
...
+		"convertible_formats": []
+	},
+	"slideshow_manifest": {
+		"readable_name": "Slideshow Manifest",
+		"multi_language": false,
+		"supplementary": false,
+		"thumbnail": false,
+		"subtitle": false,
+		"display": false,
+		"order": 3,
```
Comment (jayoshih) [inline le_utils/resources/presetlookup.json]: Could you make this order 1 instead? This is what determines priority on publish

**Pull request merged**